new-hunt-setup.md

```diff
@@ -25,7 +25,7 @@ For Discord integration (optional), you will additionally need to set up:
 
 #### Google Drive setup
 
-Cardboard expects a hunt Google Drive folder to already be set up with a template spreadsheet in it (from which puzzle spreadsheets will be cloned). Cardboard uses the list of users who have access to the Google Drive folder as the whitelist for authorizing users on login.
+Cardboard expects a hunt Google Drive folder to already be set up with a template spreadsheet in it (from which puzzle spreadsheets will be cloned). Cardboard uses the list of users who have access to the Google Drive folder as the allowlist for authorizing users on login.
```

Comment:
Second sentence can be something like:
```
Users can login to Cardboard only if they have access to the Google Drive folder.
```

or 

```
Cardboard grants login access only to users who have access to the Google Drive folder.
```

Reply:
Done. This whole paragraph is slightly inaccurate, given the recent hunt_settings changes, but perhaps this can be updated once we know how #662 will be resolved.